public/css/style.css

```diff
@@ -0,0 +1,13 @@
+@import 'icons.css';
```

Comment:
hey @afolabiopakunle, please no css file should be added to the **public** file

Reply:
If the content of the css file is for the importation of the material-icon files. If I go by this, does it mean I would be importing it into every component that needs the material icons? Note that it is from the same location that bootstrap.min.css and muli font were imported which made them central throughout the app.

Reply:
@afolabiopakunle Nope, all you need to do is import it to the **main.js** file and you can call it anywhere you want to

Reply:
Great! I'll work it out. Thanks a lot lot.